modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -0,0 +1,5471 @@
+= Cluster Configuration Properties 
+:description: Cluster configuration properties list. 
```

Comment:
```suggestion
:description: Reference of cluster configuration properties.
```

Reply:
I'll suggest a similar description in each of the property files (similar to how we describe the metrics files).